ci/openshift-ci/cluster/install_kata.sh

```diff
@@ -138,7 +138,7 @@ enable_sandboxedcontainers_extension() {
 	oc apply -f ${deployment_file}
 	oc get -f ${deployment_file} || \
 		die "Sandboxed Containers extension machineconfig not found"
-	wait_mcp_update || die "Failed to update the machineconfigpool"
+	wait_mcp_update 1200 || die "Failed to update the machineconfigpool"
```

Comment:
AFAICT this doesn't change anything as `wait_mcp_update` already defaults to 1200 :

```
wait_mcp_update() {
	local delta="${1:-1200}"
```

The `Set timeout to 900 seconds` line in https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/49597/rehearse-49597-periodic-ci-kata-containers-kata-containers-main-next-e2e-tests/1770435599786840064/artifacts/e2e-tests/kata-containers-e2e-pre/build-log.txt can only come from `wait_for_reboot()` :

```
wait_for_reboot() {
	local delta="${1:-900}"
```

If I'm missing something, please clarify some more in the commit log because `the "delta" variable is apparently set in the CI pipeline` doesn't tell much.

Reply:
> Well directly before that the "INFO: Enabling the RHCOS extension for Sandboxed Containers" is printed. Then the machine config is created and checked. After that the "Set timeout to 900 seconds" is printed and then "check" messages that come from line 128. So it really comes from the `wait_mcp_update`. Also the error message comes from mcp_update (updated vs. rebooted).
> 

Yeah you're right the messages clearly come from `wait_mcp_update()` and...

> TBH I don't know why it's not using the 1200s especially when the delta is a local variable. I can add some debug info on the `release` repo to get the environment next week.

... I now suspect the root cause is that the failing CI run mentioned above did not have 7ff2eb508e4251f5d551ba050efcd47af06b44a0.

```
 wait_mcp_update() {
-       local delta="${1:-900}"
+       local delta="${1:-1200}"
```

Can you double-check ?